src/metabase/lib/js/metadata.cljs

```diff
@@ -241,19 +241,36 @@
   (or (object-get obj "_card")
       obj))
 
-(defmethod parse-objects :card
-  [object-type metadata]
-  (let [parse-card (comp (parse-object-fn object-type) unwrap-card)]
+(defn- assamble-card
+  [metadata id]
+  (let [parse-card (comp (parse-object-fn :card) unwrap-card)]
+    ;; The question objects might not contain the fields so we merge them
+    ;; in from the table matadata.
     (merge
-     (obj->clj (comp (filter (fn [[k _v]]
-                               (str/starts-with? k "card__")))
-                     (map (fn [[s v]]
-                            (when-let [id (lib.util/string-table-id->card-id s)]
-                              [id (delay (assoc (parse-card v) :id id))]))))
-               (object-get metadata "tables"))
-     (obj->clj (comp (map (fn [[k v]]
-                            [(parse-long k) (delay (parse-card v))])))
-               (object-get metadata "questions")))))
+     (-> metadata
+         (object-get "tables")
+         (object-get (str "card__" id))
+         ;; _plainObject can contain field names in the field property
+         ;; instead of the field objects themselves.  Removing this
+         ;; property makes sure we parse the real fields.
+         (doto (gobject/remove "_plainObject"))
```

Comment:
This is the mutation I'm unsure about.

Reply:
I make a shallow copy before removing `_plainObject`.

Reply:
mentioned on Slack but I don't think you need to clone and mutate the object, I think the easiest think to do here would be to have `obj->clj` and `parse-object-fn` take an optional options map with an `ignore-plain-object?` option, then you can just have the `parse-objects` method for `:card` pass that where appropriate. Then you don't need to mutate anything

Reply:
@camsaul I've made the suggested change.